tools/upload-release

```diff
@@ -86,14 +87,18 @@ bucket.upload_file(
     ExtraArgs=extra,
 )
 
-if ordered_filenames[1] == new_basename:
-    print("Copying to zulip-server-latest.tar.gz..")
+if ordered_filenames[1] == new_basename and re.match(
+    r"^zulip-server-(\d+\.\d+)\.tar.gz$", new_basename
+):
```

Comment:
That’d mean if we release 6.0-rc1, and then release 5.8, neither will be copied to latest. We want something more like `if next(name for name in ordered_filenames if re.match(…, name)) == new_basename`.

Reply:
Ah, great point.  Repushed with that -- though apparently `natsort` returns a gnarly type, so a `cast(List[str], natsort(…))` seemed needed.  Let me know if there's a cleaner way to do that!